workflows/iss/iss.wdl

```diff
@@ -0,0 +1,134 @@
+
+task process_field_of_view {
+
+    String experiment
+    Int field_of_view
+
+    command <<<
+        python3 <<CODE
+
+        import numpy as np
```

Comment:
move this into a .py file so we can test it.

even better, but entirely optional, would be to structure it as a recipe (rationale: other people would be less tempted to tinker with the pipeline in a way that makes it non-traceable from a provenance POV).